test/parallel/test-directory-import.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+require('../common');
+const fixtures = require('../common/fixtures');
+const assert = require('assert');
+
+{
+  assert.rejects(import('./'), /ERR_UNSUPPORTED_DIR_IMPORT/);
+  assert.rejects(import(fixtures.path('packages', 'main')), /Did you mean/);
```

Comment:
Rather than importing a file path use `pathToFileURL` here - this will get the Windows tests passing.